config/spring19_seminar/cola_var/bert.conf

```diff
@@ -0,0 +1,38 @@
+// Import the defaults using the relative path
+include "../final.conf"
```

Comment:
Another comment about epochs: the defaults.conf has val_interval = 1000 and that percolates here. When training on CoLA, 2000 batches are already 3.738 epochs. Should we set a smaller number?

Reply:
Now val_interval and eval_val_interval are both set to 100